package/debian/opt/thingsplex/easee/defaults/config.json

```diff
@@ -17,5 +17,10 @@
   "pollingInterval": "30s",
   "slowChargingCurrentInAmperes": 10,
   "httpTimeout": "30s",
-  "observationsPeriod": "168h"
+  "signalR": {
+    "connCreationTimeout": "5s",
```

Comment:
Where did you get these numbers?

Reply:
Apart from `connCreationTimeout` and `invokeTimeout`, these are the defaults suggested by `philippseith/signalr`. Do you suggest to adapt it?